notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -3674,6 +3674,52 @@ father and daughter [Dr. Shinichi Chujo's adult daughter informed him that his g
 giant moth [Mothra came to Japan's defense when Godzilla attacked.]
 
 
+movie: Paycheck (2003)
+======================
+
+:: Title
+Paycheck
+
+:: Description
+Paycheck is a 2003 American science fiction action film based on the 1953
+short story of the same name by science fiction writer Philip K. Dick.
+
+Synopsis: A talented engineer accepts a contract to work on a three year
+secret project after which he will have his memory of the time erased and will
+be made rich for his efforts.
+
+:: Date
+2003-12-25
+
+:: References
+https://en.wikipedia.org/wiki/Paycheck_(film)
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+speculative prediction of the future [A main novelty of the film was a special machine that allowed people to view the future.]
+
+:: Major Themes
+facing organized pursuit [Jennings was being pursued by both the FBI and Allcom corporation goons.]
+memory erasure [Central to the plot was Jennings having his memories of the last three years erased from his mind.]
+unethical business practices [The Allcom corporation made use of classified government information to illegally build a machine to see into the future. An unnamed corporation hired Jennings to reverse engineer a competitor's technology and recreate it with improvements.]
+intellectual property in society [The viewer is made to ponder the extent to which corporations should be allowed to go to protect their intellectual property. In particular, is the practice of contractually obligating people to undergo mindwipes in the name of protecting intellectual property justified?]
+what if I had amnesia [A mindwipe left Jennings unable to remember anything that'd happened over the past three years, and he spent the film piecing together what had happened over that time span.]
+romantic love [Jennings and Rachel fell in love.]
+
+:: Minor Themes
+hologram [A group of corporate executives marveled over a small hologram of a gesticulating, beautiful woman.]
+human life choice [Jennings briefly deliberated over whether he should give up three years of life in exchange for a big payday before consenting to do so.]
+what if I suddenly came into some free money [Jennings laughed maniacally upon discovering that the shares of Allcom stock he erroneously believed himself to be in possession of were now valued at over $92 million. The film concluded with Jennings finding a lottery ticket for the winning jackpot of $90 million under the newspaper lining of a birdcage.]
```

Comment:
I think this qualifies as

https://www.themeontology.org/theme.php?name=what%20if%20I%20was%20super%20rich

unless dollars were super devaluation in this future

Reply:
Good catch. Added [what if I was super rich](https://themeontology.org/theme.php?name=what%20if%20I%20was%20super%20rich).

Reply:
I think we use one or the other of these two themes.

Reply:
Ie. not both

Reply:
Deleted the **what if I got some free money** usage, leaving only **what if I was super rich**. But I think the **what if I got some free money** note should read

"The threshold for this theme is that the amount of money is sufficiently modest that the person isn't necessarily living on easy street for the rest of their life. If more money is obtained, the theme "what if I was super rich" is used instead."

rather than

"The threshold for this theme is that the amount of money is so large that the person doesn't necessarily have to work more in their life, should they wish it. If more money is obtained, the theme "what if I was super rich" is used instead."

as it currently reads.

Reply:
Updated the **what if I suddenly got some free money** note.